app/components/elements/breadcrumb_nav_component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Elements
+  # Displays the top bread crumb navigation
+  class BreadcrumbNavComponent < ApplicationComponent
+    renders_many :breadcrumbs, BreadcrumbComponent
+
+    def initialize(dashboard: true, admin: false)
```

Comment:
When would we use `dashboard: false`? Or `admin: true`? I assume this is a seam for something coming soon.

Reply:
@astridu do you think there will be pages that don't have breadcrumbs that start with Dashboard?

Reply:
@lwrubel I don't think it's overkill. I mostly wanted to be sure I wasn't missing something already in place.

Reply:
> @astridu do you think there will be pages that don't have breadcrumbs with Dashboard as the root?

I also looked through the collections and admin pages, and I don't see any that would not start at the Dashboard. So Dashboard should be the root for all pages. Thank you!